examples/admins/locality/admin-named-locality.yaml

```diff
@@ -1,11 +1,9 @@
-id: capitolHillFeatureId
```

Comment:
Why did the id disappear here?

Reply:
it's just a git revert of my old commit, where i had added it then to highlight that there were supposed to have both id and entityId.